dap_prinz_green_jobs/analysis/ojo_analysis/process_ojo_green_measures.py

```diff
@@ -701,15 +714,13 @@ def create_agg_data(
                 "top_5_green_skills": green_skill_info,
                 "top_5_not_green_skills": not_green_skill_info,
                 # Industry
-                "num_unique_SIC2": filtered_data["SIC_2_digit"].nunique(),
-                "num_null_sic2": len(
-                    filtered_data[pd.isnull(filtered_data["SIC_2_digit"])]
-                ),
-                "num_top_sic2": filtered_data["SIC_2_digit"].value_counts()[0]
-                if len(filtered_data["SIC_2_digit"].value_counts()) > 0
+                "num_unique_SIC": filtered_data["SIC"].nunique(),
```

Comment:
i guess we should use SIC_name given the nans?

Reply:
it was actually the name which had the nan's in!